lib/app_profiler/profiler.rb

```diff
@@ -45,7 +45,7 @@ def start(params = {})
 
         clear
 
-        StackProf.start(DEFAULTS.merge(params))
+        StackProf.start(**DEFAULTS.merge, **params)
```

Comment:
The double-splat operator should be non-destructive and we will always make a copy of the first hash, so I think a merge is unnecessary here. It seems like there was a bug back then that caused the double-splat operator to modify the first hash specified, but it has been patched: https://bugs.ruby-lang.org/issues/9776. With that said, I think this is fine and could help just in case a regression happens (even though very unlikely).

Reply:
The call to merge should be removed here. It fails with ruby 2.5.